Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -43,3 +43,18 @@ public extension UIView {
         layer.rasterizationScale = UIScreen.main.scale
     }
 }
+
+// MARK: - Animate alongside keyboard
+
+public extension UIView {
+    class func animateAlongsideKeyboard(keyboardInfo: KeyboardNotificationInfo, animations: @escaping () -> Void, completion: ((Bool) -> Void)? = nil) {
+        let animationDuration = keyboardInfo.animationDuration
+        let animationOptions = keyboardInfo.animationOptions
+
+        UIView.animate(withDuration: animationDuration, delay: 0, options: animationOptions, animations: animations, completion: completion)
```

Comment:
I guess you could skip `UIView.` here, but no biggie

Reply:
You're right, I'll change it👌